core/server/data/migration/fixtures/006/01-transform-dates-into-utc.js

```diff
@@ -0,0 +1,214 @@
+var config = require('../../../../config'),
+    models = require(config.paths.corePath + '/server/models'),
+    sequence = require(config.paths.corePath + '/server/utils/sequence'),
+    moment = require('moment'),
+    lodash = require('lodash'),
+    Promise = require('bluebird'),
+    messagePrefix = 'Transforming dates to UTC: ',
+    settingsKey = '006/01',
+    _private = {};
+
+_private.getTZOffset = function getTZOffset(date) {
+    return date.getTimezoneOffset();
+};
+
+_private.getTZOffsetMax = function getTZOffsetMax() {
+    return Math.max(Math.abs(new Date('2015-07-01').getTimezoneOffset()), Math.abs(new Date('2015-01-01').getTimezoneOffset()));
+};
+
+_private.addOffset = function addOffset(date) {
+    if (_private.noOffset) {
+        return moment(date).toDate();
+    }
+
+    return moment(date).add(_private.getTZOffset(date), 'minutes').toDate();
+};
+
+module.exports = function transformDatesIntoUTC(options, logger) {
+    var ServerTimezoneOffset = _private.getTZOffsetMax(),
+        settingsMigrations = null;
+
+    return models.Base.transaction(function (transaction) {
+        options.transacting = transaction;
+
+        // will ensure updated_at fields will not be updated, we take them from the original models
+        options.importing = true;
+        options.context = {internal: true};
+
+        return sequence([
+            function databaseCheck() {
+                if (ServerTimezoneOffset === 0) {
+                    return Promise.reject(new Error('skip'));
+                }
+
+                // pg stores dates with offset, so it's enough to force timezone UTC in the db connection (see data/db/connection.js)
+                if (config.database.isPostgreSQL()) {
+                    return Promise.reject(new Error('skip'));
+                }
+
+                // sqlite stores UTC timestamps, but we will normalize the format to YYYY-MM-DD HH:mm:ss
+                if (config.database.client === 'sqlite3') {
+                    _private.noOffset = true;
+                }
+
+                return Promise.resolve();
+            },
+            function checkIfMigrationAlreadyRan() {
+                return models.Settings.findOne(lodash.extend(options, {key: 'migrations'}))
```

Comment:
@lodash will rename, but i try to follow the approach: never name a variable with one letter.
@extend yes i know saw that, in this case it will not cause any trouble, but will change so its more clear.